messaging/tests/test_api.py

```diff
@@ -289,20 +289,20 @@ def test_add_message(snapshot, project_user_api_client, project, event_selection
     snapshot.assert_match(executed)
 
 
-@pytest.mark.parametrize("protocol", ["EMAIL", "SMS"])
+@pytest.mark.parametrize("protocol", [Message.EMAIL, Message.SMS])
 @patch.object(SMSNotificationService, "send_sms")
 def test_send_email_directly_with_add_message(
     mock_send_sms, protocol, project, project_user_api_client
 ):
     ChildWithGuardianFactory()
 
     variables = get_add_message_variables(
-        project, protocol=protocol, send_directly=True
+        project, protocol=protocol.upper(), send_directly=True
     )
     project_user_api_client.execute(ADD_MESSAGE_MUTATION, variables=variables)
-    if protocol == "SMS":
+    if protocol == Message.SMS.upper():
         mock_send_sms.assert_called_once()
-    elif protocol == "EMAIL":
+    elif protocol == Message.EMAIL.upper():
         assert len(mail.outbox) == 1
```

Comment:
Why is there a need to change the case of the enums here?

```python
class Message(TimestampedModel, TranslatableModel):
    EMAIL = "email"
    SMS = "sms"
    PROTOCOL_CHOICES = ((EMAIL, _("Email")), (SMS, _("SMS")))
    protocol = models.CharField(
        max_length=16,
        verbose_name=_("notification type"),
        choices=PROTOCOL_CHOICES,
        default=EMAIL,
    )
```
and
```typescript
export enum MessagingMessageProtocolChoices {
  /** Sähköposti */
  Email = 'EMAIL',
  /** SMS */
  Sms = 'SMS'
}
```
from https://github.com/City-of-Helsinki/kukkuu-admin/pull/277/

So it'd be good to try to really understand what's going on here and why and that could this be somehow fixed in a more clean way (i.e. no explicit case handling at all). We could have a chat about this today.

Reply:
It seems Graphene is converting enum values to uppercase by default:
- https://github.com/graphql-python/graphene-django/issues/280